src/components/specific/files/document-viewer/DocumentViewer.vue

```diff
@@ -13,6 +13,8 @@ import NoDocPreviewImage from "../../../images/NoDocPreviewImage.vue";
 
 const { DWG, DXF, IFC, JPEG, PDF, PNG } = MODEL_TYPE;
 
+const OFFICE_FILES = [".ppt",".pptx",".odp",".xls",".xlsx",".ods",".doc",".docx",".odt"]
```

Comment:
This extension list could be moved into the files config: `src/config.files.js`.